### ISSUE TITLE
Create GitHub pages for our repository

### ISSUE THREAD

**Issue by pedrohyvo** [pedrohyvo/cypress-docs-pt-br#31]
Should be good to have a GitHub pages for our repository and also it will make a easier navigation between the content.

What do you think guys? @wlsf82 @lucastfa-toptal

**Comment by wlsf82** [comment]
I'm ok without it for now.
The main effort should be on the translations.

**Comment by lucastfa-toptal** [comment]
I agree, we can keep this issue and work on it later, once we have more pages translated.

**Comment by pedrohyvo** [comment]
Ok guys, let's keep this issue opened for future work while #22 is in progress .
